_tests/README.md

```diff
@@ -0,0 +1,21 @@
+# Deis Integration Tests
+
+This directory contains [Ginkgo]/[Gomega] based integration tests, which exercise
+the Deis CLI to extensively test the platform.
+
+# Resetting Cluster State
+
+Periodically, tests may not clean up after themselves. While this is an ongoing issue,
+for which we're working on a permanent fix (possible in [this GH issue](https://github.com/deis/workflow/issues/125))),
+below are commands you can run to work around the failure:
+
+```console
+$ kubectl exec -it deis-workflow-qoxhz -- sh
```

Comment:
this can be shortened:

    $ kubectl exec -it deis-workflow-qoxhz python manage.py shell